csp.config.js

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Content-Security-Policy
+ */
+const SELF = "'self'"
+const PRODUCION = '*.core.ac.uk'
+
+const config = {
+  'default-src': [SELF, PRODUCION],
+  'script-src': [SELF, '*.google-analytics.com'],
+  'style-src': [SELF],
+  // Google Analytics may transport data via image:
+  // https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#transport
+  'img-src': [SELF, PRODUCION, 'data:', '*.google-analytics.com'],
+  'connect-src': [SELF, PRODUCION, 'sentry.io', '*.google-analytics.com'],
+}
+
+if (process.env.NODE_ENV !== 'production') {
+  // Allow hot module replacement using inlined scripts and styles
+  config['script-src'].push("'unsafe-inline'", "'unsafe-eval'")
+  config['style-src'].push("'unsafe-inline'")
+
+  // Allow connection to the local hosts in development
+  // Isn't this covered by 'self'? 🤔
```

Comment:
if I remember correctly there were two problems

1. API is running on a different port
2.  when you open dashboard on localhost and API on 127.0.0.1 it's not the same.